.jenkins/pytorch/test.sh

```diff
@@ -12,9 +12,6 @@ source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 echo "Testing pytorch"
 
 if [ -n "${IN_CIRCLECI}" ]; then
-  # TODO move this to docker
-  pip_install unittest-xml-reporting coverage
-
```

Comment:
Please leave `pip_install unittest-xml-reporting` in the default clause (otherwise test status reporting is lost of all but code coverage linux config)